.babelrc

```diff
@@ -1,3 +1,13 @@
 {
-  "presets": ["react", "es2015", "stage-0"]
+  "presets": [
+    "@babel/preset-react",
+    "@babel/preset-stage-3",
+    "@babel/preset-es2017"
+  ],
+  "plugins": ["transform-class-properties"],
```

Comment:
I found this odd.

1. `transform-class-properties` is already part of `preset-stage-3`
2. tests breaks when this is omitted

Any ideas on how we pass tests without having to explicitly define this plugin in the `.babelrc`?